src/context/WalletProvider/KeepKey/helpers.ts

```diff
@@ -2,6 +2,10 @@ import type { RecoverDevice } from '@shapeshiftoss/hdwallet-core'
 import type { KeyboardEvent } from 'react'
 import { VALID_ENTROPY_NUMBERS } from 'context/WalletProvider/KeepKey/components/RecoverySettings'
 
+export const RELEASE_PAGE = 'https://github.com/keepkey/keepkey-updater/releases/latest'
```

Comment:
really should belong in config, will be easier to find/update when keepkey releases a new version of the updater app